### DISCUSSION TITLE
Planning

### DISCUSSION THREAD

**Discussion by eric-taix** [eric-taix/volley34-mobile#222] [General]
## Planning 

The current general planning is the following:
- end of 2021: all features implemented and all issues fixed
- until end of 2021: internal release (see below)
- start of January 2022: start the beta release (see below)
- mid of January 2022: start to discuss about the next release (v2)
- start of February 2022: start the public release + fix production issues
- ....

### Internal release:
  - project's members + few other people
  - can test each build (not bug free so may be instable)

### Beta release: 
  - internal release members + VCV's members + Volley34's members
  - can test ready for production releases (almost bug free)

### Public release:
  - everybody: apps are available in their respective store

**Comment by eric-taix** [comment]
@Lo0ping As you can see, we'd like to invite Volley34's members (Jerome, Emilie, and others) to test the app during the beta release (January). Can you ask them if they agree and send me their email address so I can add them as beta testers?

**Reply by Lo0ping** [reply]
I will ask them.
But not sure that they will want to report issues in English 🤣😂🤣

PS : They have Volley FSGT 34 members, not Volley 34 members 😉

**Reply by Lo0ping** [reply]
Asked...

**Comment by eric-taix** [comment]
The goal is not to let them to report issues in Github and in English ;-)
We have to find a way for beta users to report issues but I'm pretty sure they can report issues by using Google Play or Apple Store and in French of course.

BTW: Do not send me their email addresses here, but by email, as this repository is pubic!

**Reply by Lo0ping** [reply]
`We have to find a way for beta users to report issues but I'm pretty sure they can report issues by using Google Play or Apple Store and in French of course.`
try : http://bugtracker.sangalli.fr/

**Reply by eric-taix** [reply]
Well I know bugtracker and I've nothing against it but it still an issue tracker as GitHub issues is.

We won't use 2 issue trackers and for développement it's much more easier to use the one provided with the source code (links with PR, auto close feature and more).

I've tested stores comments and it's not the way to go for issue reporting (only one comment).

Perhaps we could create a WhatsApp's group: all non technical people report bugs here and technical people verify the bug and create an issue in GitHub?

**Comment by Lo0ping** [comment]
Can we plan a meeting to speak anout the projet ? 
I don't see it in your planning 😉
